### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by AxyutitsArtyom**
@AmritaKhatri
@DianaBukeikhanova

### CODE REVIEW

File: prac_08/dynamic_labels.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        return self.root
+
+    def create_widgets(self):
+        """Create labels from data and add them to the GUI."""
+        for name in self.name_to_phone:
+            # create a button for each data entry, specifying the text
+            label = Label(text=name)
+            # add the button to the "entries_box" layout widget
+            self.root.ids.main.add_widget(label)
+
```
Comment (DianaBukeikhanova) [inline prac_08/dynamic_labels.py:30]: Good working code! Nice job!

File: prac_08/convert_miles_km.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        value = self.get_valid_miles() + change
+        self.root.ids.input_miles.text = str(value)
+        self.handle_calculate()
+
+    def get_valid_miles(self):
+        try:
+            value = float(self.root.ids.input_miles.text)
+            return value
+        except ValueError:
+            return 0
```
Comment (DianaBukeikhanova) [inline prac_08/convert_miles_km.py:29]: It is optional, but you can add docstrings.

File: prac_08/convert_miles_km.kv [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+                text: 'Down'
+                on_press: app.handle_increment(-1)
+    Button:
+        text: 'Convert miles to km'
+        on_press: app.handle_calculate()
+    Label:
+        id: output_label
+        font_size: 48
+        text: '54.71756'
+        color: (1, 1, 0, 1)
```
Comment (DianaBukeikhanova) [inline prac_08/convert_miles_km.kv:26]: Good job!

File: prac_08/box_layout.kv [hunk 4]
```diff
@@ -0,0 +1,16 @@
+BoxLayout:
+    orientation: 'vertical'
```
Comment (DianaBukeikhanova) [inline prac_08/box_layout.kv:2]: Nice code

File: prac_08/dynamic_labels.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+from kivy.uix.label import Label
+
+
+class DynamicWidgetsApp(App):
+    """Main program - Kivy app to demo dynamic label creation."""
```
Comment (DianaBukeikhanova) [inline prac_08/dynamic_labels.py:8]: It should be DynamicLabelsApp

**Comment by amritakhatri** [comment]
I checked the code and its working!

**Pull request merged**